src/Network/Affjax/Request.purs

```diff
@@ -0,0 +1,115 @@
+module Network.Affjax.Request
+  ( Ajax()
+  , AjaxRequest()
+  , MethodName(..)
+  , Content(..)
+  , AjaxResponse()
+  , defaultRequest
+  , ajax
+  ) where
+
+import Control.Monad.Aff (Aff(), EffA(), makeAff)
+import Control.Monad.Eff (Eff())
+import Control.Monad.Eff.Exception(Error())
+import Data.Array ()
+import Data.ArrayBuffer.Types (ArrayView())
+import Data.Function (Fn8(), runFn8)
+import Data.Maybe (Maybe(..), maybe)
+import Data.Nullable (Nullable(), toNullable)
+import DOM (Document())
+import DOM.File (Blob())
+import DOM.XHR (FormData())
+import Network.HTTP (Header(..))
+
+-- | The event type for AJAX requests.
+foreign import data Ajax :: !
+
+-- | The parameters for an AJAX request.
+type AjaxRequest a =
+  { url :: String
+  , method :: MethodName
+  , headers :: [Header]
+  , content :: Maybe (Content a)
+  , username :: Maybe String
+  , password :: Maybe String
+  }
+
+-- | A HTTP method name: `GET`, `POST`, etc.
+newtype MethodName = MethodName String
```

Comment:
What's the problem with it? Seems the readme is a little out of date, but it's just a collection of types for the most part: https://github.com/joneshf/purescript-http/blob/master/src/Network/HTTP.purs

I only ask as I'm using the `Header` types from there as well at the moment. I don't mind re-implementing both if you'd rather I did it that way though.